README.md

```diff
@@ -43,3 +43,42 @@ Using the command console, enter the following command:
 ```git clone <link to your repository>```
 
 And Voilá! The repository is cloned to your computer!
+
+
+
+
+How to Submit a Pull Request
+
+To submit a Pull Request, follow these steps:
+
+
+•	Find a repository on Github
+
+•	Click the “Fork” button on the upper right-hand corner of the repository. What this will do is that it will save the repository to your Github account
```

Comment:
This information is covered earlier in this document, this bullet point can be removed.

Reply:
I have removed this bullet point